csxtools/utils.py

```diff
@@ -76,20 +77,28 @@ def get_fastccd_images(light_header, dark_headers=None,
 
         bgnd = np.array(dark)
 
-    data = _get_images(light_header, tag)
-    data = correct_images(data, bgnd, gain=gain)
+    data, n_images = _get_images(light_header, tag)
+    data = correct_images(data, bgnd, flat=flat, gain=gain)
     data = rotate90(data, 'cw')
-    return data
+    return data, n_images
```

Comment:
As a general comment, it's good to avoid returning tuples that don't have an obvious ordering. Users would have to remember which comes first, `n_images` or `data`.